pre_commit/commands/sample_config.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+
+# TODO: maybe `git ls-remote git://github.com/pre-commit/pre-commit-hooks` to
+# determine the latest revision?  This adds ~200ms from my tests (and is
+# significantly faster than https:// or http://).  For now, periodically
+# manually updating the revision is fine.
```

Comment:
Another option is to read https://api.github.com/repos/pre-commit/pre-commit-hooks/tags which may (or may not) be easier than invoking git.